pom.xml

```diff
@@ -19,10 +21,8 @@
 		<tag>claim-2.4</tag>
 	</scm>
 
-
 	<properties>
-		<jenkins.version>1.596.1</jenkins.version>
-		<findbugs.failOnError>false</findbugs.failOnError> <!-- TODO fix 19 bugs -->
+		<jenkins.version>2.7.3</jenkins.version>
```

Comment:
Maybe the bump is too high for somebody, but OK

Reply:
which one would you recommend ? should I stick to the 1.x versions? to a previous 2.x LTS?

Reply:
> somebody

Of course there _somebody_ on something older than that, but we won't get anywhere with anything if we consider _everyone_. [Obligatory XKCD](https://xkcd.com/1172/).

More than 90% of users of Claim Plugin are on 2.7.3 or newer, so this isn't excluding a significant number of existing users: http://stats.jenkins.io/pluginversions/claim.html

Having to update Jenkins once per year to get improvements in plugins also isn't too much to ask.